supermarq-benchmarks/examples/qre-challenge/Lessons-Learned.md

```diff
@@ -0,0 +1,98 @@
+# Architecture-Level Quantum Resource Estimator
+
+## Spencer Dearman
+
+### **Abstract**
+
+This document outlines the design and development of a architecture-level Quantum Resource Estimation (QRE) tool. Unlike existing QRE tools, which are mainly focused on quantum algorithm-level estimations, this tool aims to address the complexities involved in quantum architecture.
+
+
+### **Background and Motivation**
+
+Existing QRE tools are limited in their ability to provide insights at the architecture level, particularly when comparing Surface Codes with LRESC. Current tools just highlight the inherent advantage of LRESC in saving qubits, a conclusion that can be reached without extensive QRE analysis. This project aims to build a more advanced QRE tool that considers other factors such as Movement, Atom Reloading, Code Teleportation, Connectivity, Degree of Parallelization, and other architectural assumptions, providing more meaningful and actionable insights.
+
+ 
+### **Current Resource Estimation Limitations**
+
+_Limitations of Azure Quantum Resource Estimator and Qualtran_
+
+Both the Azure Quantum Resource Estimator (AQRE) and Qualtran are leading tools used in quantum resource estimation. However, despite their popularity, these tools are predominantly algorithm-centric. This narrow focus on algorithms presents a significant limitation when attempting to produce realistic estimations for specific and more complex quantum systems, as they largely disregard the hardware-specific characteristics that are crucial.
```

Comment:
Talk more about how they are algorithm-centric: They out-the-box assume Surface code with Lattice surgery and MSD.